torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -560,10 +560,17 @@ def rewrite_node(node):
                 return None
 
             if node.target == torch.ops.aten.expand.default:
+                input_tensor_shape = args[0].meta["val"].size()
```

Comment:
Are you aware about `ttnn.expand`?

Reply:
https://github.com/tenstorrent/tt-metal/blob/main/tests/ttnn/unit_tests/test_expand.py

Reply:
Keep in mind, I don't think its possible to convert this at 0 cost for tiles with tile padding. And because we don't have a support for a strided view - it is currently not possible at 0 cost even for Row Major tensors. 

Please try lowering to ttnn.expand.

Reply:
Thank you for remind about `ttnn.expand`.

### Test `ttnn.expand`
I try the `ttnn.expand` using https://github.com/tenstorrent/tt-metal/blob/main/tests/ttnn/unit_tests/test_expand.py, and notice that it fails to **torch.bfloat16** on certain cases due to page alignment. as below
```
function_args = (ttnn.Tensor([[ 0.53125,  0.97656,  ...,  0.00000,  0.00000],
             [ 0.00000,  0.00000,  ...,  0.00000,  0.000...0000,  ...,  0.00000,  0.00000]], shape=Shape([1[32], 3[32]]), dtype=DataType::BFLOAT16, layout=Layout::TILE), [3, -1])
function_kwargs = {}

    def __call__(self, *function_args, **function_kwargs):
>       return self.function(*function_args, **function_kwargs)
E       RuntimeError: TT_THROW @ /tmp/build-via-sdist-euf95zo8/metal_libs-0.53.0rc35+wormhole.b0/tt_metal/impl/buffers/circular_buffer_types.hpp:71: tt::exception
E       info:
E       Page size must be divisible by sizeof(uint32_t) because buffers holds uint32_t values
E       backtrace:
```

### Lowering to `ttnn.expand` in certain condition
Refer to 7aa9c4090284611cb543c31668f6f1681c8f342d, I have fix to use `ttnn.expand` with certain condition.

`ttnn.expand` can be used in the condition of **input_shape[0] %2 == 0**, test as below
```bash
$ pytest -s tests/lowering/misc/test_expand.py
```